### ISSUE TITLE
rtl8188eu does not work of the box

### ISSUE THREAD

**Issue by guysoft** [guysoft/OctoPi#206]
TP-Link and others have this, and does not seem to work on 0.12.0.
A solution is given at: https://github.com/raspberrypi/linux/issues/764

Solution is to run
```
sudo wget https://github.com/lwfinger/rtl8188eu/raw/c83976d1dfb4793893158461430261562b3a5bf0/rtl8188eufw.bin -O /lib/firmware/rtlwifi/rtl8188eufw.bin
```
Should work in nightly but did not test.

**Comment by fopina** [comment]
Proper solution is actually to use `linux-firmware`. On jessie that would mean (as `non-free` is already included in apt sources)

```
sudo apt-get install firmware-realtek
```

**Comment by guysoft** [comment]
Fixed in 0.13.0
